workshops/views.py

```diff
@@ -937,7 +937,6 @@ def all_events(request):
 
 
 @login_required
-@permission_required('workshops.add_todoitem', raise_exception=True)
```

Comment:
Yes. It's important to only process the `SimpleTodoForm` when user has `workshops.add_todoitem` permission.